Documentation/contributing.rst

```diff
@@ -1695,7 +1698,12 @@ If you intent to release a new minor release, see the
 
        make update-authors
 
-#. Generate the ``NEWS.rst`` addition:
+   .. note:: Check to see if the ``AUTHORS file`` has any formatting errors and 
+             correct them.
```

Comment:
These are the random indentation mismatches? I wonder if we should say that explicitly (and maybe also the duplicate names thing)